### ISSUE TITLE
Cannot invoke "com.google.gson.JsonObject.getAsJsonPrimitive(String)" because "text" is null

### ISSUE THREAD

**Issue by VoxLight** [ProjectBots/StreamRaidersBot#26]
![image](https://user-images.githubusercontent.com/26353962/120083901-02d5e180-c09a-11eb-8405-a2738f598705.png)

Apologies for the screenshot. But I didn't see any way to copy this text and paste it here. This error occurred on StreamRaidersBot V6.2.1beta.

Not sure what exactly happened. The bot has been running for quite a long time. I have the timer set to a large delay as well to do micro-farming throughout the day. If you need any more info please let me know how to obtain it. If there is a dump or some way to share info about the profile settings without including any of the private cookie data.

**Comment by VoxLight** [comment]
Additionally,

The bot would remain paused and stopped counting down after this error occurred. I reloaded the bot, but I am greeted with this error message:

![image](https://user-images.githubusercontent.com/26353962/120084072-614f8f80-c09b-11eb-9449-ac726bd83fae.png)

This error shows each time I try to start the bot, and will not begin farming.

**Comment by VoxLight** [comment]
Hey, so, I did manage to find the logs. But it appears that server issues were happening on StreamRaiders end. After giving it some time, everything is back working like normal. 

This issue appears to not have been caused by the bot itself. Hopefully, it can serve to better help the future development of the bot by helping to identify when server issues are present. 

closing.

[logs.app.txt](https://github.com/ProjectBots/StreamRaidersBot/files/6565300/logs.app.txt)
